### PR TITLE
Add ResNet and QA models. Created Qestion_Answering folder under examples/models

### PR DESCRIPTION
**Pull request by delphieritas**
QA model might still be tuning in the future

### CODE REVIEW

**Comment by delphieritas** [comment]
fix #4

**Comment by chrishkchris** [comment]
@delphieritas May you also add this file here, thanks https://github.com/easyfan327/rafiki_panda_dev/blob/panda-dev/examples/models/image_classification/inception.py

File: examples/models/Question_Answering/PyPandaQA.py [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+    maxIdx = np.argmax(confidence_scores)
+    confidence = confidence_scores[maxIdx]
+    best_answer = answers[maxIdx]
+    best_paragraph = paper[maxIdx]
+
+    return best_answer, confidence, best_paragraph
+
+class PyPandaQA(PandaTorchBasicModel):
+    """
+    Implementation of PyTorch Vgg
```
Comment (chrishkchris) [inline examples/models/Question_Answering/PyPandaQA.py]: it is not vgg, may change the comment

File: examples/models/Question_Answering/PyPandaQA.py [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+        confidence_scores.append(float(confidence))
+
+    maxIdx = np.argmax(confidence_scores)
+    confidence = confidence_scores[maxIdx]
+    best_answer = answers[maxIdx]
+    best_paragraph = paper[maxIdx]
+
+    return best_answer, confidence, best_paragraph
+
+class PyPandaQA(PandaTorchBasicModel):
```
Comment (chrishkchris) [inline examples/models/Question_Answering/PyPandaQA.py]: PandaTorchBasicModel was renamed to TorchModel

File: examples/models/image_classification/PyPandaResNet.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+
+KnobConfig = Dict[str, BaseKnob]
+Knobs = Dict[str, Any]
+Params = Dict[str, Union[str, int, float, np.ndarray]]
+
+
+
+
+class PyPandaResNet(PandaTorchBasicModel):
```
Comment (chrishkchris) [inline examples/models/image_classification/PyPandaResNet.py]: PandaTorchBasicModel was renamed to TorchModel

File: examples/models/Question_Answering/PyPandaQA.py (hunk 1 above)
Comment (chrishkchris) [reply examples/models/Question_Answering/PyPandaQA.py:368]: you can see the renamed class https://github.com/nusdbsystem/singa-easy/blob/master/singa_easy/models/TorchModel.py#L74

File: examples/models/Question_Answering/PyPandaQA.py (hunk 2 above)
Comment (chrishkchris) [reply examples/models/Question_Answering/PyPandaQA.py]: you can see the renamed class https://github.com/nusdbsystem/singa-easy/blob/master/singa_easy/models/TorchModel.py#L74

File: examples/models/image_classification/PyPandaResNet.py (hunk 3 above)
Comment (chrishkchris) [reply examples/models/image_classification/PyPandaResNet.py]: you can see the renamed class https://github.com/nusdbsystem/singa-easy/blob/master/singa_easy/models/TorchModel.py#L74

**Comment by nudles** [comment]
> 
> 
> @delphieritas May you also add this file here, thanks https://github.com/easyfan327/rafiki_panda_dev/blob/panda-dev/examples/models/image_classification/inception.py

no. I suggest to create a separate PR for each model instead of combining many models in a single PR. It becomes difficult to review and merge.
For this PR, if it is just for QA model, then remove the resnet model.

**Comment by chrishkchris** [comment]
> > @delphieritas May you also add this file here, thanks https://github.com/easyfan327/rafiki_panda_dev/blob/panda-dev/examples/models/image_classification/inception.py
> 
> no. I suggest to create a separate PR for each model instead of combining many models in a single PR. It becomes difficult to review and merge.
> For this PR, if it is just for QA model, then remove the resnet model.

@delphieritas Please submit the panda model of resnet and inception in other PRs, thanks

**Pull request closed without merging**